AutoPatcher/AMain.cs

```diff
@@ -24,6 +26,14 @@ public partial class AMain : Form
 
         private Thread _workThread;
 
+        private bool dragging = false;
+        private Point dragCursorPoint;
+        private Point dragFormPoint;
+
+        Config ConfigForm = new Config();
```

Comment:
this should be labelled as private and set as NULL in the global declaration - the new instance should ideally be made in the constructor